test/unit/status/pollingClientTest.ts

```diff
@@ -124,4 +125,56 @@ describe('clientTest', () => {
       expect(callCount).to.be.equal(1);
     }
   });
+
+  it('should toletare network errors', async () => {
+    const emitWarning = sandbox.spy();
+    sandbox.stub(Lifecycle, 'getInstance').returns({ emitWarning } as unknown as Lifecycle);
+    const TEST_VALUE = 'foo';
+    const pollingResultGenerator: IterableIterator<StatusResult> = generator(TEST_VALUE);
+    let callCount = 0;
+    const options: PollingClient.Options = {
+      async poll() {
+        callCount++;
+        await sleep(Duration.milliseconds(10));
+        if (callCount < 5) {
+          throw new Error('ETIMEDOUT');
+        }
+        if (callCount > 4 && callCount < 10) {
+          throw new Error('ENOTFOUND');
+        }
+        return Promise.resolve(pollingResultGenerator.next().value);
+      },
+      frequency: Duration.milliseconds(100),
+      timeout: Duration.seconds(5),
+    };
+    const client = await PollingClient.create(options);
+    const pollResult = ensureJsonMap(await client.subscribe());
+
+    expect(pollResult.name).equals('foo');
+    expect(emitWarning.callCount).to.be.equal(9 /* errors */);
+    expect(callCount).to.be.equal(9 /* errors */ + 1 /* initial poll */ + 11 /* iterations */);
```

Comment:
I tried to be descriptive here as to where the value comes from, if it's too verbose I can just put the plain number here.